src/payment/strategies/adyenv2/adyenv2-script-loader.ts

```diff
@@ -0,0 +1,60 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '../../errors';
+
+import {
+    AdyenCheckout,
+    AdyenConfiguration,
+    AdyenHostWindow
+} from './adyenv2';
+
+export default class AdyenV2ScriptLoader {
+    private _stylesheets: { [key: string]: Promise<Event> } = {};
+
+    constructor(
+        private _scriptLoader: ScriptLoader,
+        private _window: AdyenHostWindow = window
+    ) {}
+
+    load(configuration: AdyenConfiguration): Promise<AdyenCheckout> {
+        return this._loadStylesheet(
+            `https://checkoutshopper-${configuration.environment}.adyen.com/checkoutshopper/sdk/3.0.0/adyen.css`
+        ).then(() => {
+            return this._scriptLoader
```

Comment:
I believe you should be able to load both assets in parallel instead of in series. This would speed up the initialisation time. i.e.:

```ts
return Promise.all([
    this._loadStylesheet(`https://checkoutshopper-${configuration.environment}.adyen.com/checkoutshopper/sdk/3.0.0/adyen.css`),
    this._scriptLoader(`https://checkoutshopper-${configuration.environment}.adyen.com/checkoutshopper/sdk/3.0.0/adyen.js`),
])
    .then(() => {
        if (!this._window.AdyenCheckout) {
            throw new PaymentMethodClientUnavailableError();
        }

        return new this._window.AdyenCheckout(configuration);
    })
    .catch(() => {
        throw new PaymentMethodClientUnavailableError();
    });
```